release/train_tests/benchmark/train_benchmark.py

```diff
@@ -301,7 +302,8 @@ def get_metrics(self) -> Dict[str, float]:
 
         validation_time = (
             self._metrics["validation/step"].get()
-            + self._metrics["validation/iter_first_batch"].get()
+            # Exclude the time it takes to get the first batch.
+            # + self._metrics["validation/iter_first_batch"].get()
             + self._metrics["validation/iter_batch"].get()
```

Comment:
let's also report the first batch time as a separate benchmark result metric

Reply:
saw the comment on the other PR. let's make sure it's available on the dashboard.

Reply:
We can show a "time_to_first_batch" chart in the preset dashboard.